src/hooks/contextValidator.ts

```diff
@@ -31,12 +31,15 @@ export function contextNotEmptyValidator(
  * @param scriptContext the result of connecting to the context provider
  * @returns strict context avoiding null values in the type and client token
  */
-export const contextOptionClientTokenNotEmptyValidator = (
+export const optionAuthorizationNotEmpty = (
```

Comment:
I'm wondering if we can improve on the function name. Also can you update the code comment above to mention both data attributes?
```suggestion
export const validateAuthorizationData = (
```